File: submissions/2088-minimum-time-to-type-word-using-special-typewriter/solution.java
class Solution {
    public int minTimeToType(String word) {
        char initial = 'a'; 
        int count = 0;
        for(int i = 0; i < word.length(); i++){
            char current = word.charAt(i);
            int time = Math.min(Math.abs(current-initial),Math.abs(current-initial-26));
            int time2 = Math.min(time,Math.abs(current-initial+26));
            count = count+time2+1;
            initial = current;
        }
        return count;
    }
}

File: submissions/2088-minimum-time-to-type-word-using-special-typewriter/solution.js
/**
 * @param {string} word
 * @return {number}
 */
var minTimeToType = function(word) {
    let initial = 'a'; 
    let count = 0;
    for(let i = 0; i < word.length; i++){
        let current = word.charAt(i);
        let time = Math.abs(current.charCodeAt(0)-initial.charCodeAt(0));
        let time2 = Math.min(time,26-time);
        count = count+time2+1;
        initial = current;
    }
    return count;
};
